insights/specs/default.py

```diff
@@ -350,7 +350,10 @@ def dumpdev(broker):
     grub2_cfg = simple_file("/boot/grub2/grub.cfg")
     grub2_efi_cfg = simple_file("boot/efi/EFI/redhat/grub.cfg")
     grubby_default_index = simple_command("/usr/sbin/grubby --default-index")  # only RHEL7 and updwards
-    grubby_default_kernel = simple_command("/usr/sbin/grubby --default-kernel")  # RHEL6 and updwards
+    grubby_default_kernel = first_of([
+                                        simple_command("/sbin/grubby --default-kernel"),
+                                        simple_command("/usr/sbin/grubby --default-kernel")
+                                    ])
```

Comment:
@vishwanathjadhav you also need to modify the archive path in `insights.specs.insights_archive` to match this change.

Reply:
There wont be any change in the insights_archive.py
~~~
grubby_default_kernel = simple_file("insights_commands/grubby_--default-kernel")
~~

Reply:
Yes, the mangle results are `insights_commands/grubby_--default-kernel`  for both items.